waiter/project.clj

```diff
@@ -32,6 +32,8 @@
 
   :dependencies [[bidi "2.1.5"
                   :exclusions [prismatic/schema ring/ring-core]]
+                 [clj-jgit "0.8.10"
+                  :scope "test"]
```

Comment:
What effect does `:scope` have?

Reply:
doesn't put the dependency into the main waiter jar